GitFyle.Core.Api.Tests.Acceptance/Apis/Repositories/RepositoriesApiTests.Logic.cs

```diff
@@ -83,11 +91,12 @@ public async Task ShouldGetAllRepositoriesAsync()
         [Fact]
         public async Task ShouldPutRepositoryAsync()
         {
-            // given
+            // given . when
             Source randomSource = await PostRandomSourceAsync();
-            Repository modifiedRepository = await ModifyRandomRepository(sourceId: randomSource.Id);
-
-            // when
+            
+            Repository modifiedRepository =
+                await ModifyRandomRepository(sourceId: randomSource.Id);
+            
             await this.gitFyleCoreApiBroker.PutRepositoryAsync(modifiedRepository);
 
             Repository actualRepository =
```

Comment:
The PUT returns an object, no need to use GetRepositoryByIdAsync after PutRepositoryAsync 

```suggestion
            // given
            Source randomSource = await PostRandomSourceAsync();
            
            Repository modifiedRepository =
                await ModifyRandomRepository(sourceId: randomSource.Id);
            
            // when
            Repository actualRepository = await this.gitFyleCoreApiBroker.PutRepositoryAsync(modifiedRepository);
```